torch/tensortypes/nestedtensor/__init__.py

```diff
@@ -0,0 +1,45 @@
+import torch
+from . import nested
+import torch.tensortypes.nestedtensor.codegen as codegen
+import os
+
+NestedTensor = nested.NestedTensor
+
+BUILD_NESTEDTENSOR = os.getenv('BUILD_NESTEDTENSOR', 'OFF') == 'ON'
```

Comment:
I'm confused. How does one build with nested tensor on?

Reply:
Oh, it's not really a build flag if you need to use it to enable the behavior

Reply:
Yes, the reason I kept it around is, because this flag will be used in the future when I bring back the c++ extension where it is then actually a build flag.

Reply:
I don't think there's any reason for nested tensor to be a build flag. Named tensor is a build flag because it has immediate and pressing impact on the performance efficiency of core tensor operations. Nested tensor is like quantized tensor; it lives in its own world, and doesn't affect the perf of core tensor. No reason to make our build system more complicated than it needs to be by introducing yet another compile flag.

Reply:
My biggest fear is triggering the codegen on accident. This could have significant impacts on perf.